package/lib/prepareDeployment.js

```diff
@@ -30,9 +31,13 @@ module.exports = {
   },
 };
 
-const updateBucketName = (bucket, name) => {
+const updateBucket = (bucket, name, location) => {
   const newBucket = _.cloneDeep(bucket);
   newBucket.name = name;
+  if (location) {
+    newBucket.properties = newBucket.properties || {};
```

Comment:
Let's write it as `if (!newBucket.properties) newBucket.properties = {}`

Reply:
Improvement done.

Of course, I tested it ;)